web/components/feed/feed-comments.tsx

```diff
@@ -334,6 +334,9 @@ export const FeedComment = memo(function FeedComment(props: {
           />
 
           <HideableContent comment={comment} />
+          <PinnableContent comment={comment}>
+            <Col className="group"></Col>
+          </PinnableContent>
```

Comment:
These lines were just for testing right?

Reply:
uhh no haha. I don't think so?

This is the code that renders the 📌 and in the next version the "view context" link.